lib/ansible/plugins/filter/strftime.yml

```diff
@@ -1,24 +1,31 @@
 DOCUMENTATION:
   name: strftime
   version_added: "2.4"
-  short_description: date formatting
+  short_description: Returns date and/or time
   description:
-    - Using Python's C(strftime) function, take a data formatting string and a date/time to create a formatted date.
+    - Using Python's L(strftime, https://docs.python.org/3/library/datetime.html#datetime.datetime.strftime) function, take a data formatting string and a date/time to create a formatted date.
   notes:
-    - This is a passthrough to Python's C(stftime), for a complete set of formatting options go to https://strftime.org/.
+    - This is a passthrough to Python's C(datetime.datetime.strftime), for a complete set of formatting options go to https://strftime.org/.
```

Comment:
https://strftime.org/ has an incomplete set of options (because it only pulls the first table from https://docs.python.org/3/library/datetime.html#strftime-and-strptime-format-codes.) it's also not clear why this is more useful than just pointing to the official Python documentation.